### PR TITLE
chore(docs)

### PR DESCRIPTION
**Pull request by SanderElias**
Updating docs, and adding 'pricing' page

### CODE REVIEW

File: docs_extraPages/pricing.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+### always free
+</div>
+<div class="col">
+
+## Pro support
+- Everything from standartd.
+- custom OSS plugins
+- help intergrating your CMS
+- email support
+- reduced hourly fees
```
Comment (aaronfrost) [inline docs_extraPages/pricing.md:32]:
```
- Everything from __Standard__
- Custom OSS Plugins
- CMS Integration Support
- Email Support
- Preferred pricing for custom development
```

File: docs_extraPages/pricing.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  
+### contact HeroDevs
```
Comment (Jefiozie) [inline docs_extraPages/pricing.md:34]:
```suggestion
### Contact HeroDevs
```

File: docs_extraPages/pricing.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+<div class="col">
+
+## Enterprise support
+- everything from pro
+- custom private plugins
+- priority email support
+- guaranteed response times
+- fixed hourly fees
+
+### contact HeroDevs
```
Comment (Jefiozie) [inline docs_extraPages/pricing.md:45]:
```suggestion
### Contact HeroDevs
```

File: docs_extraPages/pricing.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+</div>
+</div>
+
+The fees mentioned above are based on a yearly, pre-paid subscription.
```
Comment (Jefiozie) [inline docs_extraPages/pricing.md:49]: Maybe put a disclosure here around the `price`

**Review by Jefiozie — APPROVED** [review]

File: docs_extraPages/pricing.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  - [issues](https://github.com/scullyio/scully/issues)
+- running Scully on-prem
+- all modules under the `@scullyio` npm organization
+  
+### always free
+</div>
+<div class="col">
+
+## Pro support
+- Everything from standartd.
```
Comment (geromegrignon) [inline docs_extraPages/pricing.md:28]: typo: standartd => standard

File: docs/Reference/utilities/prism-js.md [hunk 6]
```diff
@@ -9,6 +9,8 @@ position: 100
 
 ## Overview
 
+> ## NOTE: Scully supports Prims now natively, see [marked](../plugins/built-in-plugins/md.md)
```
[the remark this answers is not in the document]
Comment (SanderElias) [reply docs/Reference/utilities/prism-js.md:12]: thanks fixed

File: libs/scully/src/lib/utils/platform-server/startupSpS.ts [hunk 7]
```diff
@@ -11,71 +11,69 @@ import { readDotProperty } from '../scullydot';
...
     process.title = 'ScullyWorker';
     /** worker will pick up its in a worker and starts itself */
-    const worker = await import('./ps-worker')
-      .catch(e => {
-        console.log('worker module load error', e)
-        logError(e)
-        process.exit(16);
-      });
+    const worker = await import('./ps-worker').catch((e) => {
+      console.log('worker module load error', e);
```
Comment (Jefiozie) [inline libs/scully/src/lib/utils/platform-server/startupSpS.ts:43]: Can this line be removed?

File: scully.scully-docs.config.ts [hunk 8]
```diff
@@ -14,12 +23,18 @@ import { loadRenderer } from './scully/loadRenderer';
...
+import { join } from 'path';
+import { cpus } from 'os';
+
 const { window } = new JSDOM('<!doctype html><html><body></body></html>');
 const { document } = window;
 
-global.console.log = (first, ...args) => log(typeof first === 'string' ? first.slice(0, 120) : first, ...args);
-global.console.error = (first, ...args) => logError(String(first).slice(0, 60));
-
+// global.console.log = (first, ...args) => log(typeof first === 'string' ? first.slice(0, 120) : first, ...args);
```
Comment (Jefiozie) [inline scully.scully-docs.config.ts:36]: THis can all be removed

File: scully/loadRenderer.ts [hunk 9]
```diff
@@ -6,21 +6,20 @@ export const loadRenderer = async () => {
...
-    .argv;
-  logOk('Selected', rnd)
+    .describe('rnd', 'determine how 404 is handled').argv;
+  // logOk('Selected', rnd);
   switch (rnd) {
     case 'ppt':
-      await import('@scullyio/scully-plugin-puppeteer')
-      logOk('Using Puppeteer to render pages')
+      await import('@scullyio/scully-plugin-puppeteer');
+      // logOk('Using Puppeteer to render pages');
```
Comment (Jefiozie) [inline scully/loadRenderer.ts:14]: remove the commented lines

**Review by Jefiozie — APPROVED** [review]
LGTM, small comments

**Review by Jefiozie — APPROVED** [review]

**Pull request merged**